antarest/study/storage/variantstudy/model/command/create_xpansion_candidate.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2025, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+from typing import Any, Dict, List, Optional, Tuple
+
+from typing_extensions import override
+
+from antarest.study.business.model.xpansion_model import XpansionCandidateInternal
+from antarest.study.storage.rawstudy.model.filesystem.config.model import FileStudyTreeConfig
+from antarest.study.storage.rawstudy.model.filesystem.factory import FileStudy
+from antarest.study.storage.variantstudy.model.command.common import (
+    CommandName,
+    CommandOutput,
+)
+from antarest.study.storage.variantstudy.model.command.icommand import ICommand
+from antarest.study.storage.variantstudy.model.command.xpansion_common import (
+    assert_candidate_is_correct,
+    assert_xpansion_candidate_name_is_not_already_taken,
+)
+from antarest.study.storage.variantstudy.model.command_listener.command_listener import ICommandListener
+from antarest.study.storage.variantstudy.model.model import CommandDTO
+
+
+class CreateXpansionCandidate(ICommand):
+    """
+    Command used to create a xpansion candidate
+    """
+
+    # Overloaded metadata
+    # ===================
+
+    command_name: CommandName = CommandName.CREATE_XPANSION_CANDIDATE
+
+    # Command parameters
+    # ==================
+
+    candidate: XpansionCandidateInternal
+
+    @override
+    def _apply_config(self, study_data: FileStudyTreeConfig) -> Tuple[CommandOutput, Dict[str, Any]]:
+        return CommandOutput(status=True, message="ok"), {}
+
+    @override
+    def _apply(self, study_data: FileStudy, listener: Optional[ICommandListener] = None) -> CommandOutput:
+        candidates_obj = study_data.tree.get(["user", "expansion", "candidates"])
```

Comment:
I think we should use a pydantic model to validate what we receive from the file study tree